startserver.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+for command in pypy python python3.9 python3.8 python3.7 python3  
+do
+
+#!/bin/bash
+currentver=$($command -c "import platform;print(platform.python_version())")
+# echo $currentver
+requiredver="3.7.9"
+ if [ "$(printf '%s\n' "$requiredver" "$currentver" | sort -V | head -n1)" = "$requiredver" ]; then 
+        echo "python version ok ($currentver)"
+        $command pymine
+        exit 0
+ fi
+done
+
+echo "Couldnt find Suitable python version. please use your package manager to install python 3.7.9 or later"
```

Comment:
There's a missing `'` in `Couldnt`, the `S` in `Suitable` needs to be lowercase, and the `p` and `Please` should be capitalized. Also, `python` should be `Python`.